src/python/WMArchive/PySpark/RecordAggregator.py

```diff
@@ -51,7 +51,7 @@ def extract_stats(record, timeframe_precision="day"):
                 acquisitionEra = output.get('acquisitionEra')
                 if acquisitionEra is not None:
                     break
-        if exitCode is None:
+        if exitCode is None or exitCode == '99999' or exitCode = 99999:
             for error in step['errors']:
                 exitCode = str(error.get('exitCode'))
```

Comment:
BTW, if exitCode has None value, it will be casted to `None`, which will evaluate to `True` in the next if.